### PR TITLE
Modify CVE-2021-41145 for GHSA-jvpq-23v4-gp3m

### PR DESCRIPTION
**Pull request by shelbyc**
Modify CVE-2021-41145 for GHSA-jvpq-23v4-gp3m

### CODE REVIEW

File: 2021/41xxx/CVE-2021-41145.json [hunk 1]
```diff
@@ -1,18 +1,85 @@
 {
```
Comment (cve-team) [inline 2021/41xxx/CVE-2021-41145.json]: 'ASSIGNER' is a required property

Comment (cve-team) [inline 2021/41xxx/CVE-2021-41145.json]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/41xxx/CVE-2021-41145.json]: 'product_name' is a required property

Comment (cve-team) [inline 2021/41xxx/CVE-2021-41145.json]: 'version_value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by ktodd229** [comment]
Please review the necessary changes, fix, and resubmit.

File: 2021/41xxx/CVE-2021-41145.json (hunk 1 above)
Comment (shelbyc) [reply 2021/41xxx/CVE-2021-41145.json]: Pushed modified json file that should correct the issue.

Comment (shelbyc) [reply 2021/41xxx/CVE-2021-41145.json]: Pushed modified json file that should correct the issue.

Comment (shelbyc) [reply 2021/41xxx/CVE-2021-41145.json]: Pushed modified json file that should correct the issue.

**Comment by anthnysingleton** [comment]
Thank you for your submission. Expect CVE-2021-41145 to be published on http://cve.mitre.org in the next few hours.

**Pull request merged**